stacks/dhis2-core/helmfile.yaml

```diff
@@ -100,6 +100,65 @@ releases:
           requests:
             cpu: {{ requiredEnv "RESOURCES_REQUESTS_CPU" }}
             memory: {{ requiredEnv "RESOURCES_REQUESTS_MEMORY" }}
+      {{- if eq (requiredEnv "STORAGE_TYPE") "filesystem" }}
+      - initContainers:
+          - name: "{{ requiredEnv "INSTANCE_NAME" }}-init-fs"
+            image: alpine:3.18
+            env:
+              - name: HOSTNAME
+                value: {{ requiredEnv "HOSTNAME" }}
+              - name: DATABASE_ID
+                value: "{{ requiredEnv "DATABASE_ID" }}"
+              - name: IM_ACCESS_TOKEN
+                value: {{ requiredEnv "IM_ACCESS_TOKEN" }}
+            command:
+              - "/bin/sh"
+              - "-c"
+              - |
+                # Create necessary directories
+                mkdir -p /dhis-home/files
```

Comment:
Reminder to use `DHIS2_HOME` here and below.

Reply:
Actually this is in the init container and the volume is mounted to /dhis-home, so this is correct